remember.go

```diff
@@ -0,0 +1,97 @@
+package remember
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+type Memory struct {
+	input []io.Reader
+	output io.ReadWriter
+}
+
+type Option func(*Memory) error
+
+func WithInput(args []string) Option {
+	return func(m *Memory) error {
+		var input = []io.Reader{}
+		for _, str := range args {
+			input = append(input, bytes.NewBufferString(str))
+		}
+		m.input = input
+		return nil
+	}
+}
+
+func WithOutput(output string) Option {
+	return func(m *Memory) error {
+		m.output = bytes.NewBufferString(output)
+		return nil
+	}
+}
+
+func NewMemory(opts ...Option) (Memory, error) {
+	_, err := os.Stat("store.txt")
+	if err != nil {
+		os.Create("store.txt")
+	}
+	defaultFile, err := os.OpenFile("store.txt", os.O_APPEND|os.O_RDWR, 0644)
+	m := Memory{
+		[]io.Reader{os.Stdin},
+		defaultFile,
+	}
+	for _, opt := range opts {
+		err := opt(&m)
+		if err != nil {
+			return Memory{}, err
+		}
+	}
+	return m, nil
+}
+
+func (m *Memory) Recall() string {
+	data, err := ioutil.ReadAll(m.output)
```

Comment:
After Go version 1.15, `ReadAll` is now in the `io` package. [io.ReadAll documentation](https://pkg.go.dev/io/ioutil#ReadAll) states "As of Go 1.16, this function simply calls io.ReadAll."